cli/args/flags_net.rs

```diff
@@ -32,6 +32,7 @@ pub fn validator(host_and_port: &str) -> Result<String, String> {
   if Url::parse(&format!("internal://{host_and_port}")).is_ok()
     || host_and_port.parse::<IpAddr>().is_ok()
     || host_and_port.parse::<BarePort>().is_ok()
+    || NetDescriptor::parse(host_and_port).is_ok()
```

Comment:
Can we just do this instead of the `Url::parse` stuff?

Reply:
maybe the first two could be removed? not 100% sure, don't really wanna drive by validation changes in this pr.